src/services/createGroups.js

```diff
@@ -0,0 +1,25 @@
+import shuffle from './shuffel'
+import randomSeed from 'random-seed'
+import _ from 'lodash'
+
+const rand = randomSeed.create()
+
+export default function createGroup(data) {
+  let { names, size = 4, seed = 'Web Development' } = data
+
+  if (names === '') return // early out
+
+  names = names?.split(',').map((item) => item.trim())
+  shuffle(names, rand, seed)
+  let groups = _.chunk(names, Number(size))
+  const lastGroup = groups[groups.length - 1]
+
+  if (lastGroup.length === 1 && groups.length > 1) {
+    groups = [
+      ...groups.slice(0, groups.length - 2),
+      groups[groups.length - 2].slice().concat(lastGroup),
+    ]
+  }
```

Comment:
What happens when you have groups of 4 and only two students are in the last group?

Reply:
Good question! Thanks for pointing that out!